### ISSUE TITLE
Change netcode to a more scalable solution.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mrthuanvn/wot-xvm#138]
```
Current netcode pounds server very hard. Every query needs to be decoded and 
broken down into separate entries, reply is ~6KB. Timeouts and 500 errors are 
very common.

I propose changing from web server to DNS. http://tools.ietf.org/html/rfc1464

pros:
- automatically cached by all DNS servers around the world. only first query 
from particular ISP will hit the server, following queries will stop at local 
ISP server until DNS record TTL is reached
- build in timeout mechanism
- sub 20ms response times
- removes the need to store cache locally on the HDD

cons:
- requires 30 queries in quick succession (shouldnt be a problem)


Basically store every players record in a separate subdomain. Client will query:
rasz_pl.domain.com
another_player.domain.com
and_another_player.domain.com

and so on for all 30 players. Replies will mostly come from local ISP DNS 
caches. Set TTL to 24 hours and you reduce server load to one query per player 
field per ISP per day max.
```

Original issue reported on code.google.com by `citiz...@gmail.com` on 2 Sep 2012 at 7:52
